russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography.csl

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="ru-RU" et-al-min="4" et-al-use-first="1">
+  <info>
+    <title>Russian GOST R 7.0.5-2008 (numeric-citations, alphabetical-bibliography)</title>
+    <title-short>GOST R (numeric-citations, alphabetical-bibliography)</title-short>
+    <id>http://www.zotero.org/styles/russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography</id>
+    <link href="http://www.zotero.org/styles/russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography" rel="self"/>
+    <link href="http://protect.gost.ru/document.aspx?control=7&amp;id=173511" rel="documentation"/>
+    <link href="http://standartgost.ru/g/%D0%93%D0%9E%D0%A1%D0%A2_%D0%A0_7.0.5-2008" rel="documentation"/>
+    <author>
+      <name>Rybakov Dmitriy</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>Russian GOST-2008 style. Book, webpage and article chapters edited. Note!!! In articles form of № = Issue (Volume).</summary>
+    <updated>2015-06-27T09:14:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", " suffix=",">
+      <label prefix=" " suffix=" "/>
+      <name initialize-with="."/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author" delimiter="," prefix=" / " suffix=",">
+      <name et-al-use-first="5" initialize-with="."/>
```

Comment:
OK, you'd want to properly define this then as 
```et-al-min="6" et-al-use-first="5"``` 
just using et-al-use-first without an corresponding et-al-min leaves this unclear, not sure what other processors will do with it, even though I think it works in Zotero/Mendeley.